anhaidgroup/py_entitymatching
# Enhan rule based matcher

Rule-Based Matcher
- Coding
    1. Update __init__.py to import BooleanRuleMatcher
    2. Update booleanrulematcher.py to add rule based matching and pass test cases
    3. Add parse_conjunct to generic_helper.py in utils. Have both rule based matcher and blocker use it.
- Testing
    1. Added test_rule_based_matcher.py
- Documentation
    1. Update matchers.rst with ML sections and RB section. See blocking.rst for example.
    2. Update supported_matchers.rst with ML section and RB section
- Jupiter Notebook
    1. Create notebook for RB Matcher

Triggers
- Coding
    1. Update __init__.py to import MatchTrigger
    2. Update matchtrigger.py to add triggers and pass test cases
- Testing
    1. Added test_match_trigger.py
- Documentation
    1. Add triggers.rst
- Jupiter Notebook
    1. Create notebook for triggers
